packages/scandipwa/src/component/ProductActions/ProductActions.container.js

```diff
@@ -326,7 +342,27 @@ export class ProductActionsContainer extends PureComponent {
         } = customPrice;
 
         const selectedOptions = this.getSelectedOptions();
-        const prices = options.reduce((acc, { data = [] }) => {
+        const selectedOptionsMulti = this.getSelectedOptionsMulti();
+
+        const prices = options.reduce((acc, { data = [], option_id, type }) => {
+            /*
+            * Such types contain a single item within data
+            * as those are looked up on the option_id
+            */
+            const types = ['area', 'field', 'file'];
+            if (types.includes(type)) {
+                if (selectedOptionsMulti.includes(option_id)) {
+                    if (data[0].price_type === 'PERCENT') {
```

Comment:
Please assign `data[0]` to some variable with a verbose name so that its meaning is clear

Reply:
Shipped comment to reduce code lines :innocent: